source/reference/minio-mc/mc-support-inspect.rst

```diff
@@ -46,7 +46,7 @@ Exercise caution before sending a report to a third party or posting the report
 Wildcards
 ---------
 
-The command supports wildcard ``*`` pattern matching for prefixes or objects. 
+The command supports wildcard ``*`` pattern matching for prefixes or objects when using the Bash shell. 
```

Comment:
🤔 do we know what it does for others?

Reply:
Yep, it displays a message that wildcard patterns are only supported in Bash.
Added this to the doc.